react/SelectBox/SelectBox.jsx

```diff
@@ -60,6 +60,11 @@ const reactSelectControl = CustomControl => ({
   </div>
 )
 
+export const withPrefix = (cx, className) => {
+  const classNameWithPrefix = cx(null, { [` ${className}`]: true })
+  return classNameWithPrefix === '' ? className : classNameWithPrefix
```

Comment:
This condition is `true` only when `className` is an empty string, no ? If yes, I don't understand its purpose.

Reply:
Actually no, since there's a space in ` ${className}`. I don't understand how this condition can be `true` :thinking:

Reply:
classNameWithPrefix is empty when we use `react-select` without `classNamePrefix`